kolibri/plugins/my_downloads/assets/src/views/DownloadsList/PaginatedListContainerWithBackend.vue

```diff
@@ -0,0 +1,132 @@
+<template>
```

Comment:
This file has been copied from https://github.com/learningequality/kolibri/blob/develop/kolibri/plugins/facility/assets/src/views/PaginatedListContainerWithBackend.vue - I think we'd be better off moving that component into `kolibri-common` and then just reusing it here rather than duplicating.

Reply:
Perfect, I'll do it!